src/markdown-pages/install-with-kurl/cis-compliance.md

```diff
@@ -0,0 +1,68 @@
+---
+path: "/docs/install-with-kurl/cis-compliance"
+date: "2021-12-14"
+weight: 24
+linktitle: "CIS Compliance"
+title: "CIS Compliance"
+isAlpha: false
+---
+The kURL installer can be configured to be Center for Internet Security (CIS) compliant. This is an opt-in feature that is configured in the kURL specification by setting the `kurl.cisCompliance` field to `true`. For information about known limitations, see [Known Limitations](#known-limitations). For more information about CIS security compliance for Kubernetes, see the [CIS benchmark information](https://www.cisecurity.org/benchmark/kubernetes).
+
+The following settings are changed when `cisCompliance` is set to `true`:
```

Comment:
Active voice recommendation:

When you set `cisCompliance` to `true`, kURL changes the following settings: 

Also, I'm lacking background so of course ignore if this isn't relevant, but I'm wondering if "changes" is the right verb here. Do these settings already have values that are being changed? Or, is the installer setting some of the values for the first time?

Reply:
Resolved in https://github.com/replicatedhq/kurl.sh/pull/763